configs/setups/awicm3/awicm3.yaml

```diff
@@ -672,7 +808,7 @@ oasis3mct:
                                         - 'A_CurX:A_CurY <--gauswgt_i-- u_feom:v_feom'
                 'v3.2':
                         version: 6.0-YAC
-                        mct_version: 4.0
+                        mct_version: 6.0
```

Comment:
Is this correct? Bumping up the mct_version of the `v3.2`?

Reply:
Yes, v3.2 was with oasis 6.0 and yac. in 3.2.2 i went back to 4.0.

Reply:
But then again this never really worked. So it should not be used. too many people use v3.2 over 3.2.2. I will revert it.

Reply:
i also deleted the line about using version 6.0-YAC above..

Reply:
Perhaps should be tested with v3.2

Reply:
I'll test v3.2